mutations/createWorkspace.ts

```diff
@@ -0,0 +1,17 @@
+import { UserWorkspace } from '@/models/user';
+import { gql } from '@apollo/client';
+
+export interface CreateWorkspaceMutationResponse {
+  createWorkspace: UserWorkspace & { __typename?: 'Workspace' };
+}
+
+export const CREATE_WORKSPACE_MUTATION = gql`
+  mutation CreateWorkspace($name: String!) {
+    createWorkspace(name: $name) {
+      id
```

Comment:
why is there an id here? this is stating what is going to come in the response right?

Reply:
Yeah, and I want the id so I can route to `/workspaces/${id}/` once the newly created workspace has been added